apis/spark/src/main/java/io/tiledb/vcf/VCFInputPartitionReader.java

```diff
@@ -280,6 +281,11 @@ private void initVCFReader() {
                 + " MB.");
       }
 
+      if (bufferSizeMB > 2048) {
+        log.warn("Size of individfaual buffers is larger than 2048 MB. Consider setting a smaller " +
```

Comment:
Put a note that this is a limitation of arrow 0.10.0 in spark 2.4, so people know. 

Also we need to take some action here. Should we error out or should we adjust the buffer now to 2048 max? What do you think the user would expect?

Also we should do the math for the user and tell them what is the max size they can set.

Reply:
I believe that we should adjust it to 2048. I also refactored the warning message.